### ISSUE TITLE
High severity vulnerability found by Snyk scanner - Denial of Service (DoS)

### ISSUE THREAD

**Issue by pavol-antalik-mox** [kubernetes-sigs/aws-ebs-csi-driver#1800]
/kind bug

**What happened?**

Snyk scan in our CI pipeline revealed vulnerabilities in the [1.24.0 release image](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/tag/v1.24.0):

```
✗ High severity vulnerability found in google.golang.org/grpc
  Description: Denial of Service (DoS)
  Info: https://security.snyk.io/vuln/SNYK-GOLANG-GOOGLEGOLANGORGGRPC-5953328
  Introduced through: google.golang.org/grpc@v1.57.0
  From: google.golang.org/grpc@v1.57.0
  Fixed in: 1.56.3, 1.57.1, 1.58.3

✗ High severity vulnerability found in golang.org/x/net/http2
  Description: Denial of Service (DoS)
  Info: https://security.snyk.io/vuln/SNYK-GOLANG-GOLANGORGXNETHTTP2-5953327
  Introduced through: golang.org/x/net/http2@v0.15.0
  From: golang.org/x/net/http2@v0.15.0
  Fixed in: 0.17.0
```

More info:
https://security.snyk.io/vuln/SNYK-GOLANG-GOOGLEGOLANGORGGRPC-5953328
and
https://security.snyk.io/vuln/SNYK-GOLANG-GOLANGORGXNETHTTP2-5953327

**What you expected to happen?**

Please, update packages to above suggested versions with fix.

**Comment by ConnorJC3** [comment]
We're in the process of releasing versions 1.23.2 and 1.24.1 which will contain updated dependencies with a fix for these CVEs.

**Comment by torredil** [comment]
Hi @pavol-antalik-mox, thank you for reporting this. The CVEs mentioned above have been fixed in our latest releases: `1.23.2` and `1.24.1`.

/close

**Comment by pavol-antalik-mox** [comment]
Thank you, our build now finishes the Snyk check without issues.
